src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-1n + 2),
+.stars--3 :nth-child(-1n + 3),
+.stars--4 :nth-child(-1n + 4),
+.stars--5 :nth-child(-1n + 5) {
```

Comment:
The use of `:nth-child(-1n + X)` is unconventional and can be confusing. A more common approach would be to use `:nth-child(n+X)`, which is easier to understand. For example, `.stars--1 :nth-child(n+1)` would target the first child and so on. This change would improve readability and maintainability of your CSS.